### ISSUE TITLE
I think RequestWithDcMigration needs timeout

### ISSUE THREAD

**Issue by artygrand** [sochix/TLSharp#869]
Forever loop caused, if phone without country code passed to SendCodeRequestAsync, for example.

**Comment by knocte** [comment]
Please send a pull-request.

**Comment by knocte** [comment]
Wait a sec, is the forever-loop happening inside TLSharp?

**Comment by artygrand** [comment]
Yes. If telegram returns DataCenterMigrationException from every DC, request cannot be completed

**Comment by knocte** [comment]
But there should be a way to detect the infinite loop without using a timeout.
